packages/dotcom-ui-base-styles/src/lib/fontFaces.ts

```diff
@@ -1,6 +1,8 @@
+// NOTE: please also update ./styles.css when updating the URLs defined below. They
```

Comment:
i'm sure i've written a package at some point to extract variables from sass. can't find it right now. so, it's possible, but i think this is fine for now

Reply:
oh, i didn't write it, but i have used it before https://github.com/XOP/sass-vars-to-js